### PR TITLE
LPS-185796 Add span with sr-only so that the screen reader correctly reads the node name

### PR DESCRIPTION
**Pull request by veroglez**
This fix has to do with this task: https://liferay.atlassian.net/browse/LPS-185796

Currently when the screen reader reaches the node it reads something like the following:
`Select container move container container hide container options open`

With this change, only the name of the node is read.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTree.js [hunk 1]
```diff
@@ -424,6 +426,10 @@ export default function PageStructureSidebar() {
 									<ClayTreeView.Item
 										actions={<ItemActions item={item} />}
 									>
+										<span className="sr-only">
+											{item.name}
+										</span>
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTree.js:431]: Can't we add this to the StructureTreeNode component?

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:forward:force

**Pull request closed without merging**